notebooks/116-sparsity-optimization/116-sparsity-optimization.ipynb

```diff
@@ -8,7 +8,9 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    with open(f"{quantized_sparse_dir}/perf_config.json", "w") as outfile:
</pre><p>with (qunatized_sparse_dir / "perf_config.json".open( "w") as outfile: </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/903/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
changed in https://github.com/openvinotoolkit/openvino_notebooks/pull/903/commits/aaf8b124245124bb8e2f5f97d5bed158c1e7118e )